### PR TITLE
Specification, see commit msg

### PR DESCRIPTION
**Pull request by albertmink**
Note: lines with multiple sentences are bad for code-review, L118

### CODE REVIEW

File: doc/specification.md [hunk 1]
```diff
@@ -23,16 +23,16 @@ with the mandatory elements object_name, object_type and file_extension.
 Namespaces are converted into brackets, for instance `/NMSPC/CL_OBJECT` becomes `(nmspc)cl_object`.
 
 ### Object Name and Type
-The object name and type represent the ABAP object. Basically, the object name (R3TR) and type are stored in the ABAP object directory (aka TADIR).
+The object name (R3TR) and object type are derived from the object itself, as stored in the ABAP object directory (aka TADIR).
 
 ### Sub Object Name and Sub Object Type
 
-Some object types support sub objects (LIMU object types) like function modules in a function group. For sub objects, the main object name and type must be part of the file name.
+Some objects support sub objects (LIMU object) like function modules in a function group. For sub objects, the main object name and type must be part of the file name.
```
Comment (schneidermic0) [inline doc/specification.md:30]: I am not sure whether we should stick to `object types` here.

File: doc/specification.md [hunk 2]
```diff
@@ -80,7 +80,7 @@ The JSON files additionally follow
 
 ## Format Compatibility
 
-The format of object types will evolve over time. If a change to the format is considered incompatible, the version of the file format has to be increased.
+The format of objects will evolve over time. If a change to the format is considered incompatible, the version of the file format has to be increased.
```
Comment (schneidermic0) [inline doc/specification.md:83]: I am not sure whether we should stick to `object type` here.

**Review by schneidermic0 — COMMENTED** [review]
Thanks, Albert. Removing ABAP in front of objects is fine. I am not sure whether we should stick to the term "object type" if we are talking about types and not objects.

File: doc/specification.md (hunk 1 above)
Comment (albertmink) [reply doc/specification.md]: Reverted to the previous version.

File: doc/specification.md (hunk 2 above)
Comment (albertmink) [reply doc/specification.md]: Reverted to the previous version.

**Review by schneidermic0 — APPROVED** [review]
Thanks. LGTM :)

**Pull request merged**